config/nova/interactions/mount_anal.json

```
@@ -0,0 +1,30 @@
+{
+	"name": "Mount (Anal)",
+	"description": "Put your ass on their cock.",
+	"usage": "other",
+	"category": "‏Anal",
+	"distance_allowed": 0,
+	"message": [
+    	"%USER% impales herself with %USER_PRONOUN_THEIR% anal hole on %TARGET%'s sturdy shaft.",
```

```suggestion
    	"%USER% anally impales %USER_PRONOUN_THEM%self with %TARGET%'s sturdy shaft.",
```
Doesn't really sound right the way you had it also no hardcoded pronouns!